packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -30,24 +80,29 @@ const isUnSupportedContract = (portfolio_pos: TIsUnSupportedContract) =>
     !!portfolio_pos.is_forward_starting; // for forward start contracts
 
 export const formatPortfolioPosition = (
-    portfolio_pos: TPortfolioPos,
+    portfolio_pos: ProposalOpenContract,
     active_symbols: TActiveSymbols = [],
     indicative?: number
 ) => {
     const purchase = portfolio_pos.buy_price;
     const payout = portfolio_pos.payout;
-    const display_name = getSymbolDisplayName(active_symbols, getMarketInformation(portfolio_pos.shortcode).underlying);
-    const transaction_id =
-        portfolio_pos.transaction_id || (portfolio_pos.transaction_ids && portfolio_pos.transaction_ids.buy);
+    const display_name = getSymbolDisplayName(
+        active_symbols,
+        getMarketInformation(portfolio_pos.shortcode ?? '').underlying
+    );
+    // const transaction_id =
```

Comment:
why did you change this:
`        portfolio_pos.transaction_id || (portfolio_pos.transaction_ids && portfolio_pos.transaction_ids.buy);`

Reply:
> why did you change this: ` portfolio_pos.transaction_id || (portfolio_pos.transaction_ids && portfolio_pos.transaction_ids.buy);`

It's because type 'ProposalOpenContract' doesn't have property 'transaction_id', only 'transaction_ids'.